imod/prepare/wells.py

```diff
@@ -25,21 +25,42 @@ def vectorized_overlap(bounds_a, bounds_b):
     )
 
 
+def compute_point_filter_overlap(bounds_wells, bounds_layers):
+    """
+    Special case for filters with zero filter length, these are set to layer
+    thickness.
+    """
+    zero_filter_length = bounds_wells[:, 1] == bounds_wells[:, 0]
```

Comment:
Please start a boolean variable with "is" or "has" ->` is_zero_length_filter` or `has_zero_filter_length`

Reply:
Improved to ``has_zero_filter_length``.

Reply:
Improved to ``has_zero_filter_length``.